AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/booking-status/booking-status.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, Input } from '@angular/core';
+import { BookingsDetailsModel } from '../../common/model/bookings-list.model';
+@Component({
+    selector: 'app-booking-status',
+    templateUrl: './booking-status.component.html',
+    styleUrls: ['./booking-status.component.css']
+})
+export class BookingStatusComponent {
+    @Input() bookingDetails: BookingsDetailsModel;
```

Comment:
In the chance the whole model changes, this input would change too. It would be safer to have the input be status of type HearingStatus. That way you could even re-use this component in the booking details page and pass in the status irrespective of the model for that page

Reply:
the booking detail page and booking list page status' are a little differnt. The detail page isn't supposed to show the 'no judge' badge. We're also using the booking detail model proprties to select the css elements. Also the start time and duration from the model are displayed in this component view (because the sit conveniently between the NO JUDGE badge and the status text

Reply:
Deleted my previous comment, you're right I'm not actually using it anywhere anymore. Ill remove it

Reply:
We're using other properties from the BookingDetailsModel, such as the time and duration and the flags for the status? Are you saying we should make a model for the input? The status property on the BookingDetailsModel is just a string. Also this status component is slightly differnt to the one shown in the booking detail view, as that one doesnt show the no judge badge.